src/mbgl/gl/texture.hpp

```diff
@@ -8,6 +8,18 @@ namespace gl {
 
 class Texture {
 public:
+    Texture(Size size_, UniqueTexture texture_,
+            TextureFilter filter_ = TextureFilter::Nearest,
+            TextureMipMap mipmap_ = TextureMipMap::No,
+            TextureWrap wrapX_ = TextureWrap::Clamp,
+            TextureWrap wrapY_ = TextureWrap::Clamp)
+        : size(std::move(size_)),
+          texture(std::move(texture_)),
+          filter(filter_),
+          mipmap(mipmap_),
+          wrapX(wrapX_),
+          wrapY(wrapY_) {}
```

Comment:
Yes, I already did this for some, but need to do another check on all classes/structs affected :+1: